test/fixtures/cookbooks/test/recipes/filter_lines_replace_between.rb

```diff
@@ -0,0 +1,27 @@
+#
+# Verify the results of using the replace_between filter
+#
+
+directory '/tmp'
+
+# ==================== replace_between filter =================
+insert_lines = %w(line1 line2 line3)
+start_pattern = /COMMENT/
+end_pattern = /main/
+
+template '/tmp/replace_between' do
+  source 'dangerfile.erb'
```

Comment:
We probably don't want to rely on this file being the same. 

@Xorima and I have a habit of changing it at a whim

Reply:
Well speaking of changing it, I have some ideas to run past you at some point ... plus I kinda want to get rid of it somehow at some point ...

Reply:
I plan to change the name of dangerfile soon, since it is not the dangerfile used for danger checking.  This dangerfile was there long before we implemented dangerfile checking.

Reply:
I replaced dangerfile.erb to avoid confusion with Dangerfile.